### ISSUE TITLE
Wk 2 summaries

### ISSUE THREAD

**Issue by Castlevania1997** [stefanbund/cis1-a#2]
There is a overwhelming amount of articles but not what I'm looking for or require the purchase of the article. In some, it's too big to summarize.

**Comment by linkjankens** [comment]
Having the same problem, one in particular was 700 pages long. Also everything I've found needs to be purchased. The abstract is available on some of them though; would it be okay to just summarize that?

**Comment by stefanbund** [comment]
Yes! Simply knowing that that knowledge is out there will be valuable to us

Stefan

**Comment by stefanbund** [comment]
Many articles do you to be purchased. 

However, you should steer clear of those, as no one is required to buy anything. 

The abstract may be valuable, however. To do the best job, go ahead and choose articles that are viewable in your browser, but do not require payment. Those listings which indicate they are .pdf types will read completely without a fee.

Stefan
